Population.cpp

```diff
@@ -884,9 +876,37 @@ int Population::transfer(Landscape* pLandscape, short landIx)
 			}
 		}
 	}
+	return ndispersers;
+}
+
+// Transfer is run for populations in the matrix only
+#if RS_RCPP // included also SEASONAL
+int Population::transfer_settle(Species* pSpecies, std::vector<Individual*>& inds, Landscape* pLandscape, short nextseason)
```

Comment:
```suggestion
int Population::transfer_settle(Species* pSpecies, std::vector<Individual*>& dispInds, Landscape* pLandscape, short nextseason)
```
same as above, also need to rename all calls to `inds` in this function